cmd/nerdctl/builder_build_test.go

```diff
@@ -45,7 +45,6 @@ CMD ["echo", "nerdctl-build-test-string"]
 	base.Cmd("build", buildCtx, "-t", imageName).AssertOK()
 	base.Cmd("run", "--rm", imageName).AssertOutExactly("nerdctl-build-test-string\n")
 
-	// DOCKER_BUILDKIT (v20.10): `Error response from daemon: exporter "docker" could not be found`
 	if base.Target == testutil.Nerdctl {
```

Comment:
The comment above should be kept, unless we can remove this base.Target check too

Reply:
Fair point.
Let me see about that.

Reply:
Base target check removed on latest push (works locally with docker 27 - pending CI).